versioning/BRM/extentmap.cpp

```diff
@@ -5272,19 +5007,18 @@ void ExtentMap::getSysCatDBRoot(OID_t oid, uint16_t& dbRoot)
     bool bFound = false;
     grabEMEntryTable(READ);
     grabEMIndex(READ);
-    int emEntries = fEMShminfo->allocdSize / sizeof(struct EMEntry);
 
-    for (int i = 0 ; i < emEntries; i++)
+    for (auto emIt = fExtentMapRBTree->begin(), end = fExtentMapRBTree->end(); emIt != end; ++emIt)
```

Comment:
I presume that syscat will be in the beginning of the tree so it is safe to left this iteration loop.